src/nuxt-plugin.ts

```diff
@@ -11,12 +11,15 @@ const PlausiblePlugin: Plugin = (context, inject) => {
     domain: optionsDomain.length ? optionsDomain : null,
     hashMode: optionsHashMode === 'true',
     trackLocalhost: optionsTrackLocalhost === 'true',
-    apiHost: optionsApiHost.length ? optionsApiHost : 'https://plausible.io'
+    apiHost: optionsApiHost.length ? optionsApiHost : 'https://plausible.io',
+    ...context.$config?.plausible
```

Comment:
Do we really need this? Options are always passed from the module and runtimeConfig is added there.